resources/js/visual-editor/blocks/spacer/spacer.css

```
@@ -0,0 +1,58 @@
+.wp-block-spacer {
+    clear: both;
+}
+
+/* editor.scss */
+.block-editor-block-list__block[data-type="artisanpack/spacer"]::before {
+    content: "";
+    display: block;
+    position: absolute;
+    z-index: 1;
+
+    width: 100%;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)